### ISSUE TITLE
android input  work  error

### ISSUE THREAD

**Issue by GodGavin** [cogentcore/core#1392]
### Describe the bug

https://github.com/user-attachments/assets/45d21f22-2aea-49b4-bf1a-f9c81dda1e85

I build the example/demo and it's input can't input
See the video 

### How to reproduce

core build android

### Platform

Android

**Comment by GodGavin** [comment]
it's a very bad bug👿

**Comment by kkoreilly** [comment]
Thank you for reporting this issue. We will work on fixing it soon.

**Comment by GodGavin** [comment]
> Thank you for reporting this issue. We will work on fixing it soon.
Thans.
And it's can't input Chinese words in input 
window，the cursor is not on the right position it’s outside the inputbox
![Screenshot_20241224-141129](https://github.com/user-attachments/assets/d27aba93-b150-476a-aa1e-f419b45a184e)

**Comment by kkoreilly** [comment]
Yes, we are aware of the Chinese character rendering issue (#568), which we are planning to fix within the next few weeks. We will also look into the cursor position issue. Thank you again for reporting these issues.
